andela/ah-the-answer-backend
# #164829360 email verification

__What does this PR do__
Verifies a user email

__Description of tasks to be completed__
- Configure email settings
- Create email HTML template
- Create verify email view

__How should this be manually tested__
- Clone the repo
 `git clone https://github.com/andela/ah-the-answer-backend.git `
- Create a virtual environment and activate it.
        -`virtualenv venv`
        -`Source env/bin/activate`
- Export sendgrid email config to your environment.
  
- Run local tests
        - `python manage.py run tests` 
- The endpoint can be tested through Postman as follows.
        - Run server `python manage.py runserver`
        - Navigate to `POST http://127.0.0.1:8000/api/users/` on postman.
        - Enter the following payload on the body 
              ```  {
	    "user":{
		    "email":"valide_mail",
		     "password":"password",
		      "username":"username"
	              }
             } ```
         - U should get back the details you have provided as the output.
         - Check the email you have provided for a cerification email
         - Click the link to verify your account

__Relevant PT stories___
https://www.pivotaltracker.com/story/show/164829360
